demos/tutorials/6_fluid_tank.ipynb

```diff
@@ -0,0 +1,296 @@
+        "\n",
+        "The fluid tank is a cylinder of radius $R_\\mathrm{tank}$ and height $h_\\mathrm{tank}$, initially filled with a height $h_\\mathrm{fluid}$ of fluid. New fluid is injected with velocity $v_\\mathrm{inlet}$ into the tank from a circular inlet at the top of the container with arbitrary coordinates $(x_\\mathrm{inlet}, y_\\mathrm{inlet})$. The fluid exits the tank from a cylindrical outlet at the bottom of the container.\n",
+        "\n",
+        "In this tutorial, we perform simulations of this scenario varying a single parameter, in order to demonstrate the impact of design parameters in a metric of interest.\n",
+        "\n",
+        "More specifically, we vary $x_\\mathrm{inlet}$ and show that it has an important impact on the average fluid velocity inside the tank. We thus illustrate that Inductiva API can be used for streamlined experimentation with simulations, including in optimization pipelines."
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "JifnwkPumJ0D"
+      },
+      "source": [
+        "## Setup\n",
+        "\n",
+        "We start by installing and importing the Inductiva API package. We also import other packages required for this tutorial."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ANM9oLopONBQ"
+      },
+      "outputs": [],
+      "source": [
+        "# Install the Inductiva API package.\n",
+        "!pip install inductiva"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 2,
+      "metadata": {
+        "id": "kIFfuKXfOhfO"
+      },
+      "outputs": [],
+      "source": [
+        "# Import Inductiva API package.\n",
+        "import inductiva\n",
+        "\n",
+        "# Import also other packages needed for this tutorial.\n",
+        "import numpy as np\n",
+        "import xarray as xr\n",
+        "import matplotlib.pyplot as plt"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "metadata": {
+        "id": "MoYtO0bYOlru"
+      },
+      "outputs": [],
+      "source": [
+        "# Set the API key to be able to access our hardware.\n",
```

Comment:
Can you add this link "https://forms.gle/MkcvWA8BuUFmGVVu6" and mention that the key can be requested here.